addons/mail/static/src/models/activity/activity.js

```diff
@@ -256,6 +268,14 @@ function factory(dependencies) {
         }),
         category: attr(),
         creator: many2one('mail.user'),
+        /**
+         * Determines that an activity is linked to a requesting partner or not.
+         * It will be used notably in website slides to know who triggered the
+         * "request access" activity.
+         * Also, be useful when the assigned user is different from the
+         * "source" or "requesting" partner.
+         */
+        requestingPartner: many2one('mail.partner'),
```

Comment:
alphabetical order: `requestinPartner` between `note` and `state`

Reply:
Done!